src/Config.hpp

```diff
@@ -0,0 +1,278 @@
+#ifndef __DVL_TELEDYNE_CONFIG_HPP_
+#define __DVL_TELEDYNE_CONFIG_HPP_
+
+#include <base/Time.hpp>
+#include <dvl_teledyne/PD0Messages.hpp>
+#include <math.h>
+
+namespace dvl_teledyne
+{
+
+
+enum BAUDRATE
+{
+    BR300,
+    BR1200,
+    BR2400,
+    BR4800,
+    BR9600,
+    BR19200,
+    BR38400,
+    BR57600,
+    BR115200
+};
+
+enum PARITY
+{
+    NONE = 1,
+    EVEN,
+    ODD,
+    LOW,
+    HIGH
+};
+
+enum SENSORSOURCE
+{
+    MANUAL,
+    INTERNAL,
+    EXTERNAL
+};
+
+/**DVL_teledyne configuration.
+ * Holds selected settings to be sent to the DVL.
+ * Initialized with factory defaults.
+ * For a list of all configuration commands and detailed information of the
+ * ones used here, please read the ExplorerDVL Operation Manual.
+ * NOTE: Setting marked with e_ are considered expert settings and should be
+ * used with caution and after consulting the  manual.
+ */
+struct Config
+{
+    Config()
+    : bottomTrackPingsPerEnsemble(1)
```

Comment:
Just a recommendation, but since you're switching the package to C++11, it would IMO be nicer to use the field initialization syntax, e.g.

~~~
int maximumTrackingDepth = 100;
~~~